crates/basilica-sdk-python/python/basilica/sandbox.py

```
@@ -0,0 +1,1358 @@
+            self.refresh()
+
+            if self.state == SandboxState.READY:
+                return
+            if self.state == SandboxState.FAILED:
+                raise SandboxError(f"Sandbox failed to start: {self.sandbox_id}")
+            if self.state == SandboxState.TERMINATED:
+                raise SandboxError(f"Sandbox was terminated: {self.sandbox_id}")
+
+            time.sleep(poll_interval)
+
+        raise SandboxError(f"Timeout waiting for sandbox to become ready")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)